Cargo.toml

```diff
@@ -1,6 +1,11 @@
 [workspace]
 default-members = ["apollo-router", "apollo-router-core"]
-members = ["apollo-router", "apollo-router-core", "apollo-router-benchmarks", "xtask"]
+members = [
+    "apollo-router",
+    "apollo-router-core",
+    "apollo-router-benchmarks",
+    "xtask",
+]
```

Comment:
1. :yarn: You keep changing that and someone else keep changing it back.
2. :screwdriver: This file is unnecessarily touched in this PR.